Competitive Programming/c2cpp.md

```diff
@@ -0,0 +1,117 @@
+# C to Cpp for competitive
+
+This article is written for specifically keeping competetive in mind and not for using cpp in development or any other context. Also this aims at getting started to use good and important features of c++ as quickly as possible so skips many details.
+Most of the c++ fetures talked here work on c++98 and after but some work only after c++11 so for simplicity we will assume to be using c++11. Also we will be using gcc/g++ compiler. You can also use an IDE like codeblocks it already has gcc and g++ compilers by default (MINGW); remember to enable c++11 or c++14 option in codebloks' settings.
```

Comment:
`s/talked/mentioned`
`s/after/above`
`s/after c++11/on c++11 and above`
`s/codebloks/Codeblocks`

Add full-stops between sentences.

Reply:
Some people might not know what `c++98` and `c++11` mean. I think you should explain that C++ has seen many language updates over time, which has lead to various versions of C++. Perhaps we could put this information in the main `CP1.md` article.